.github/workflows/source_aws.yml

```diff
@@ -47,12 +47,26 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 2
-      - name: Set up Go 1.x
+      - name: Set up Go
+        id:   set-up-go
         uses: actions/setup-go@v3
         with:
-          go-version-file: plugins/source/aws/go.mod
-          cache: true
+          go-version-file:       plugins/source/aws/go.mod
+          cache:                 true
```

Comment:
Should we disable the `setup-go` caching if we're doing it manually? Also can we drop the indentation changes?

Reply:
setup-go does the dependency (go mod) cache, which we omit in the build & test cache. Basically, both are required

Reply:
What's the value of having separate caches? It's seems that having them separated means duplicate save/restore operations, also separate cache keys (we should verify that actually otherwise they'll override each other)

Reply:
It just requires additionally to track the go mod dir location & I'd rather leave it to the Go team =)

Reply:
Between the tradeoff of having two different caches I'd rather have the hard coded value as that's not likely to change and also shown in the cache action docs, see https://github.com/actions/cache/blob/04f198bf0b2a39f7230a4304bf07747a0bddf146/examples.md#linux-1

Reply:
I left only the setup-go cache for now